public/app/percona/shared/components/PerconaBootstrapper/PerconaBootstrapper.tsx

```diff
@@ -36,7 +40,9 @@ export const PerconaBootstrapper = ({ onReady }: PerconaBootstrapperProps) => {
   const { user } = config.bootData;
   const { isSignedIn } = user;
   const theme = useTheme2();
+  const { userId } = useSelector(getPerconaUser);
 
+  const [modalNewVersionShown, setModalNewVersionShown] = useLocalStorage(`${userId}-grafana.pmm3.modalShown`, false);
```

Comment:
You could probably use just `config.bootData.user.id` for getting the ID of the user without tapping into the store.